src/libs/actions/Session.js

```diff
@@ -212,7 +219,11 @@ function resetPassword() {
  * Restart the sign in process by clearing everything from Onyx
  */
 function restartSignin() {
+    const environment = currentEnvironment;
```

Comment:
Not something we need to tackle in this PR, but just spit-balling, I like the idea of have different subclasses of Onyx keys:

1. Keys that actually aren't persisted in `AsyncStorage` – allow Onyx to be used as a regular-old event bus
2. Regular keys that can be evicted safely
3. Regular keys that shouldn't be evicted unless we have no choice
4. "invincible" keys that won't disappear when we call `Onyx.clear()`

This would be one of those "invincible" keys 🙂